spec/models/manageiq/providers/redfish/physical_infra_manager/refresher_spec.rb

```diff
@@ -57,4 +59,13 @@ def assert_physical_server_details
       :resource_type => "PhysicalServer"
     )
   end
+
+  def assert_hardwares
+    h = Hardware.first
+    expect(h).to have_attributes(
+      :disk_capacity   => 0,
```

Comment:
might be nice to add some disks to the test server?

Reply:
Test server reported no disks when I recorded the session, so capacity is 0. And yes, we really need a better testbed for Redfish, but let us not go there;)